tuskyapp/Tusky
# Add Mastodon Redirect support

Mastodon Redirect is my app that handles redirecting Mastodon links to supported apps, so the apps don't have to maintain their own list of supported instances.

This PR adds auto-detect support for Mastodon Redirect, so Tusky can be chosen for opening Mastodon links.

To test:
1. Download Mastodon Redirect from https://github.com/zacharee/MastodonRedirect.
2. Install Tusky from this PR's code.
3. Open and log into Tusky.
4. Find a random post and copy its link. Paste the link somewhere it'll be clickable, such as in Telegram's "Saved messages" channel.
5. Remove Tusky from Recents.
6. Open Mastodon Redirect and select Tusky in the "Discovered" section.
7. Tap the pasted link. It should launch Tusky and then resolve to the post.
8. Back out to the main feed in Tusky and go back to the link, without removing Tusky from Recents this time. The link should open Tusky and resolve to the post.
